src/main/kotlin/com/simplymerlin/minigameserver/minigame/oitc/OneInTheChamberGame.kt

```diff
@@ -0,0 +1,84 @@
+package com.simplymerlin.minigameserver.minigame.oitc
+
+import com.simplymerlin.minigameserver.Server
+import com.simplymerlin.minigameserver.core.Minigame
+import com.simplymerlin.minigameserver.core.phase.EndPhase
+import com.simplymerlin.minigameserver.core.state.ScheduledStateSeries
+import com.simplymerlin.minigameserver.minigame.oitc.phase.GamePhase
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.format.NamedTextColor
+import net.kyori.adventure.text.format.TextDecoration
+import net.minestom.server.MinecraftServer
+import net.minestom.server.coordinate.Point
+import net.minestom.server.coordinate.Pos
+import net.minestom.server.entity.Player
+import net.minestom.server.instance.InstanceContainer
+import net.minestom.server.instance.batch.AbsoluteBlockBatch
+import net.minestom.server.instance.block.Block
+import net.minestom.server.item.Material
+
+class OneInTheChamberGame(instance: InstanceContainer, server: Server) : Minigame(instance, server) {
+    override val name
+        get() = "One In The Chamber"
+    override val displayName = Component.text("One In The Chamber", NamedTextColor.RED, TextDecoration.BOLD)
+    override val displayDescription = listOf(Component.text("Beat your friends in some classic OITC!", NamedTextColor.RED))
```

Comment:
this looks ugly af

Reply:
i am not a designer!

Reply:
well I just think we should change this when we brand it